rma/views/rma_portal_templates.xml

```diff
@@ -262,6 +269,10 @@
                 <h2>Communication</h2>
                 <t t-call="portal.message_thread">
                     <t t-set="object" t-value="rma"/>
+                    <t t-set="object" t-value="rma"/>
+                    <t t-set="token" t-value="rma.access_token"/>
```

Comment:
Do we have "Share" button in the backend?

Reply:
Yes

Reply:
duplicated line `<t t-set="object" t-value="rma"/>`

Reply:
Fixed. Thanks @kos94ok-3D